docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
```

Comment:
I like the use of the box here to demarcate the boundaries of the sequence activation! Really helps to improve clarity